openshift/rh-che.app.yaml

```diff
@@ -0,0 +1,316 @@
+                name: che
+          - name: CHE_FABRIC8_USER__SERVICE_ENDPOINT
+            valueFrom:
+              configMapKeyRef:
+                key: che-fabric8-user-service-endpoint
+                name: che
+          - name: CHE_FABRIC8_WORKSPACES_ROUTING__SUFFIX
+            valueFrom:
+              configMapKeyRef:
+                key: che-fabric8-workspace-routing-suffix
+                name: che
+          - name: CHE_DOCKER_ENABLE__CONTAINER__STOP__DETECTOR
+            valueFrom:
+              configMapKeyRef:
+                key: che-docker-enable-container-stop-detector
+                name: che
+          - name: CHE_WORKSPACE_CHE__SERVER__ENDPOINT
+            valueFrom:
+              configMapKeyRef:
+                key: che-workspace-che-server-endpoint
+                name: che
+          - name: CHE_WORKSPACE_CHE_LOGS
+            valueFrom:
+              configMapKeyRef:
+                key: che-workspace-logs
+                name: che
+          - name: CHE_KEYCLOAK_AUTH__SERVER__URL
+            valueFrom:
+              configMapKeyRef:
+                key: che-keycloak-auth-server-url
+                name: che
+          - name: CHE_KEYCLOAK_REALM
+            valueFrom:
+              configMapKeyRef:
+                key: che-keycloak-realm
+                name: che
+          - name: CHE_KEYCLOAK_CLIENT__ID
+            valueFrom:
+              configMapKeyRef:
+                key: che-keycloak-client-id
+                name: che
+          - name: CHE_HOST
+            valueFrom:
+              configMapKeyRef:
+                key: che-host
+                name: che
+          image: ${IMAGE}:${IMAGE_TAG}
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            httpGet:
+              path: /api
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 120
+            timeoutSeconds: 10
+          name: che
+          ports:
+          - containerPort: 8080
+            name: http
+          - containerPort: 8000
+            name: http-debug
+          readinessProbe:
+            httpGet:
+              path: /api
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 15
+            timeoutSeconds: 60
+          resources:
+            limits:
+              memory: 600Mi
+            requests:
+              memory: 256Mi
+          volumeMounts:
+          - mountPath: /data
+            name: che-data-volume
+        serviceAccountName: che
+        volumes:
+        - name: che-data-volume
+          persistentVolumeClaim:
+            claimName: che-data-volume
+    triggers:
+    - type: ConfigChange
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: che
+    name: che
+  spec:
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+      kind: Service
+      name: che-host
+parameters:
+- name: IMAGE
+  value: rhche/che-server-multiuser
```

Comment:
I think default image should be the one we push to `registry.devshift.net` not from dockerhub

Reply:
Not sure if it really matters, since it will be overridden anyway. In che-starter dockerhub is the default one [1] 

[1] https://github.com/ibuziuk/che-starter/blob/master/openshift-template/che-starter.app.yaml#L109

Reply:
Image for prod / prod-preview is defined in dsaas-services:
https://github.com/openshiftio/saas-openshiftio/blob/master/dsaas-services/che-starter.yaml#L6